x-pack/plugins/security_solution/public/common/components/inspect/modal.tsx

```diff
@@ -153,22 +135,7 @@ export const ModalInspectQuery = ({
       ),
       description: (
         <span data-test-subj="index-pattern-description">
-          <p>{formatIndexPatternRequested(inspectRequests[0]?.index ?? [])}</p>
-
-          {!isSourcererPattern && (
```

Comment:
hey i just put this in! https://github.com/elastic/kibana/pull/125007

What gives?

Reply:
I must have done a crap job of pulling changes from the other branch. sorry about that! I'll restore your changes and try adding back in the intended set of changes

Reply:
@stephmilovic Take another look! I think I removed all the mistaken changes